test/test_v2/test_clientapp_funcs.c

```diff
@@ -223,3 +223,29 @@ void pmixt_validate_predefined(pmix_proc_t *myproc, const pmix_key_t key,
         }
     }
 }
+
+double avg_fence_time(void) {
+	double avg_fence = 0.0;
+	int i, retval;
+    unsigned long usecs;
+    struct timeval local_start, local_end;
+
+    // Synchronize before timing
+    if (0 != PMIx_Fence(NULL, 0, NULL, 0)) {
+        TEST_ERROR_EXIT(("PMIx_Fence Problem: ret val = %d", retval));
+    }
+	/* Measure the average typical fence execution time */
+    gettimeofday(&local_start, NULL);
+	for(i = 0; i < 100; i++) {
+        if (0 != PMIx_Fence(NULL, 0, NULL, 0)) {
+            TEST_ERROR_EXIT(("PMIx_Fence Problem: ret val = %d", retval));
+        }
+	}
+    gettimeofday(&local_end, NULL);
+    usecs = (local_end.tv_sec * 1000000) + local_end.tv_usec
+        - ((local_start.tv_sec * 1000000) + local_start.tv_usec);
+    avg_fence = ((double)usecs)/1E8;
+
+    TEST_VERBOSE(("Simple PMIx_Fence, avg time: %lf", avg_fence));
+	return avg_fence;
+}
```

Comment:
The mix of tabs and spaces here bugs me. Can you clean that up? In general, we prefer spaces not tabs.

Reply:
@jjhursey  Sure and good catch, I don't really know how that happened. I'm using VS Code and it should be defaulting to spaces. I'll fix it and look into why it arose.